StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/ReturnedFromFinancialConnections.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ReturnedFromFinancialConnections.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 10/17/24.
+//
+
+/// Notifies that the user finished the FinancialConnections flow
+struct ReturnedFromFinancialConnections: MessageSender {
+    struct Payload: Codable, Equatable {
+        /// The linked bank account token.
+        /// This value will be nil if the user canceled the flow or an error occurred.
+        let bankToken: String?
```

Comment:
Alternatively, if we don't need to return the bank token, we could send a `success | cancelled | failed` enum value to the web view